priv/perf/README.md

```diff
@@ -2,14 +2,63 @@
 
 Umbrella app for performance/load/stress tests
 
-## How to run the test
-1. Change the urls inside the [config files](config/) for the environment you are targeting.
-1. To generate the open-api client (only need once): `make init`
-1. To run the test: `MIX_ENV=<your_env> mix test`. Or `mix test` if you want to run against local services.
+## How to run the tests
 
-### Spin up local services for development
-1. To test with local services (by docker-compose): `make start-services`
-1. To turn the local services down: `make stop-services`
+### 1. Set up the environment vars
+
+```
+export CHILD_CHAIN_URL=<childchain api url>
+export WATCHER_INFO_URL=<watcher-info api url>
+export ETHEREUM_RPC_URL=<ethereum node url>
+export CONTRACT_ADDRESS_PLASMA_FRAMEWORK=<address of the plasma framework contract>
+export CONTRACT_ADDRESS_ETH_VAULT=<address of the eth vault contract>
+export CONTRACT_ADDRESS_ERC20_VAULT=<address of the erc20 vault contract>
+export LOAD_TEST_FAUCET_PRIVATE_KEY=<faucet private key>
+```
+
+
+### 2. Generate the open-api client
+ ```
+make init
+```
+(You need to run this any time you change the env vars above)
```

Comment:
I don't think this is true. And if any change that causes this we should try to see if we can avoid that. The new `LoadTest.Connection.XXX` is designed to not need to `make init` with url changes.

Reply:
Ah yes, you're right - this was fixed